ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.0__4.7.yaml

```diff
@@ -0,0 +1,166 @@
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: operator-unit-test-coverage
+  postsubmit: true
+  steps:
+    test:
+    - as: unit-coverage
+      commands: HOME=$(pwd) make test submit-coverage
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: operator-e2e
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable
+      OO_INSTALL_NAMESPACE: openshift-adp
+      OO_PACKAGE: oadp-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: e2e
+      commands: CLUSTER_PROFILE=aws make test-e2e
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test-oadp-operator
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 512Mi
+    workflow: optional-operators-ci-aws
+- as: operator-e2e-aws-periodic-slack
+  cron: 0 4 * * *
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable
+      OO_INSTALL_NAMESPACE: openshift-adp
+      OO_PACKAGE: oadp-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: e2e
+      commands: CLUSTER_PROFILE=aws make test-e2e
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test-oadp-operator
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 512Mi
+    workflow: optional-operators-ci-aws
+- as: operator-e2e-gcp-periodic-slack
+  cron: 0 4 * * *
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable
+      OO_INSTALL_NAMESPACE: openshift-adp
+      OO_PACKAGE: oadp-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: e2e
+      commands: CLUSTER_PROFILE=gcp make test-e2e
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test-oadp-operator
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 512Mi
+    workflow: optional-operators-ci-gcp
+- as: operator-e2e-azure-periodic-slack
+  cron: 0 4 * * *
+  steps:
+    cluster_profile: azure4
+    dependencies:
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable
+      OO_INSTALL_NAMESPACE: openshift-adp
+      OO_PACKAGE: oadp-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: e2e
+      commands: CLUSTER_PROFILE=azure make test-e2e
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test-oadp-operator
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 512Mi
+    workflow: optional-operators-ci-azure
+zz_generated_metadata:
+  branch: master
```

Comment:
Like https://github.com/openshift/release/pull/27668/files#diff-60fc118d92d4e178610b009a1d62ab4db3d3c705dd9dba56b3b8ced85a40b2caR158 , shouldn't this be `oadp-1.0`?